setup.py

```diff
@@ -17,6 +17,6 @@
     extras_require={
         'dotenv': ['python-dotenv'],
         'yaml': ['pyyaml'],
-        'async': ['asgiref >= 3.2'],
+        'async': ['asgiref >= 3.2', 'flask[async]>=2.0.0'],
```

Comment:
I think it's no need to pin the Flask version. If the user wants to use the async feature, they will use Flask 2.